spacy/tests/pipeline/test_morphologizer.py

```diff
@@ -197,3 +199,29 @@ def test_overfitting_IO():
     gold_pos_tags = ["NOUN", "NOUN", "NOUN", "NOUN"]
     assert [str(t.morph) for t in doc] == gold_morphs
     assert [t.pos_ for t in doc] == gold_pos_tags
+
+
+def test_store_activations():
+    # Simple test to try and quickly overfit the morphologizer - ensuring the ML models work correctly
```

Comment:
The docstring here is copied from another test but should read something else (also for other tests of other components)

Reply:
Fixed.